community/modules/compute/schedmd-slurm-gcp-v6-nodeset/main.tf

```diff
@@ -102,3 +102,23 @@ locals {
 data "google_compute_default_service_account" "default" {
   project = var.project_id
 }
+
+# tflint-ignore: terraform_unused_declarations
+data "google_compute_reservation" "reservation" {
+  count   = var.reservation_name != "" ? 1 : 0
+  name    = var.reservation_name
+  project = var.project_id
+  zone    = var.zone
+
+  lifecycle {
+    postcondition {
+      condition     = self.self_link != null
+      error_message = "couldn't find the reservation ${var.reservation_name}}"
+    }
+
+    postcondition {
+      condition     = coalesce(self.specific_reservation_required, true)
+      error_message = "your reservation has to be specific"
```

Comment:
This statement is vague and should be improved.

Reply:
This is not a well known term and it does not show in the UI. Specific is a very general term.

We could link to https://cloud.google.com/compute/docs/instances/reservations-overview#how-reservations-work.

Maybe just:
`Your reservation has to be specific, see https://cloud.google.com/compute/docs/instances/reservations-overview#how-reservations-work for more information.`

Reply:
Also it would be useful to add something like this "if it's intentionally automatic, don't specify it the blueprint"

Reply:
Updated.